validation/predeploy-proxyadmin_test.go

```diff
@@ -0,0 +1,63 @@
+package validation
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum-optimism/optimism/op-bindings/predeploys"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testProxyAdminIsAdminOfPredeploysForChain(t *testing.T, chain ChainConfig) {
+	// Create an ethclient connection to the specified RPC URL
+	client, err := ethclient.Dial(chain.PublicRPC)
+	require.NoError(t, err, "Failed to connect to the Ethereum client at RPC url %s", chain.PublicRPC)
+	defer client.Close()
+
+	proxyAdminAddr := predeploys.ProxyAdminAddr
+
+	proxyAdmin, err := bindings.NewProxyAdmin(proxyAdminAddr, client)
+	require.NoError(t, err)
+
+	getProxyAdmin := func(a common.Address) (common.Address, error) {
+		return proxyAdmin.GetProxyAdmin(&bind.CallOpts{}, a)
+	}
+
+	for k, p := range predeploys.Predeploys {
```

Comment:
This increases our dependence on the op-bindings package, which we want to avoid #280